ocaml/xapi/xapi_tunnel.mli

```diff
@@ -20,6 +20,7 @@ val create :
   -> transport_PIF:[`PIF] Ref.t
   -> network:[`network] Ref.t
   -> protocol:API.tunnel_protocol
+  -> cross_server:bool
```

Comment:
I think we should make the `cross_server` parameter optional and with default value as `false` for backward compatibility, then XCP-ng's usage of the internal tunnel will not break.